bruno/collections/Rafiki/environments/Local Playground.bru

```diff
@@ -14,4 +14,12 @@ vars {
   apiSignatureVersion: 1
   apiSignatureSecret: iyIgCprjb9uL8wFckR+pLEkJWMB7FJhgkvqhTQR/964=
   assetId: USD
+  gfranklinWalletAddress: https://cloud-nine-wallet-backend/accounts/gfranklin
+  bhamchestWalletAddress: https://cloud-nine-wallet-backend/accounts/bhamchest
+  wbdcWalletAddress: https://cloud-nine-wallet-backend/accounts/wbdc
+  pfryWalletAddress: https://happy-life-bank-backend/accounts/pfry
+  planexWalletAddress: https://happy-life-bank-backend/accounts/planex
+  asmithWalletAddress: https://happy-life-bank-backend/accounts/asmith
+  larsWalletAddress: https://happy-life-bank-backend/accounts/lars
+  davidWalletAddress: https://happy-life-bank-backend/accounts/david
```

Comment:
if they dont seem useful, we can also just remove these entirely from here and remove the `brunoEnvVar` in the seed files

Reply:
I guess im not quite following... these are the wallet address id's that we are setting in `loadWalletAddressIdsIntoVariables` right? Why would we even need to do `loadWalletAddressIdsIntoVariables` ? I would prefer the explicitness of setting them here but I'm probably missing something (harder to handle all environments, like rafiki money, or??)

Reply:
The walletAddressIds are the local UUIDs for the wallet addresses in each corresponding rafiki instance. It would (probably) be nice to be able to use walletAddressUrls instead of the UUIDs in Admin API mutations, but as we have it now, it's what we have to do